esp/scm/ws_topology.ecm

```diff
@@ -316,6 +316,15 @@ ESPStruct TpClusterNameType
     bool   IsDefault;
 };
 
+ESPStruct TpStoragePlan
```

Comment:
should be "Planes" not "Plan" here and elsewhere.

Reply:
Plane or Planes?